### DISCUSSION TITLE
Signing in student developer pack

### DISCUSSION THREAD

**Discussion by Tayyab-R** [community/community#38252] [GitHub Education]
Hey community, 
I have my university mail but without student identity card I am unable to get student developer pack. I have Microsoft Azure student account is there any way to get github student developer with the help of Microsoft Azure student account.
Guidance will be appreciated
Thanks in advance

**Comment by samus-aran** [comment]
Hi @Tayyab-R 

You may still apply to Global Campus without Student ID. Other forms of evidence is accepted depending on the institution you apply for. Proof of evidence accepted (out with Student ID) can be current transcript, a letter on school letterhead, or any documentation with a date that demonstrates your current enrolment 👍
